diff_drive_controller/src/diff_drive_controller.cpp

```diff
@@ -511,33 +586,39 @@ namespace diff_drive_controller{
     for (int i = 0; i < twist_cov_list.size(); ++i)
       ROS_ASSERT(twist_cov_list[i].getType() == XmlRpc::XmlRpcValue::TypeDouble);
 
-    // Setup odometry realtime publisher + odom message constant fields
+    /// Setup odometry message constant fields
     odom_pub_.reset(new realtime_tools::RealtimePublisher<nav_msgs::Odometry>(controller_nh, "odom", 100));
     odom_pub_->msg_.header.frame_id = "odom";
     odom_pub_->msg_.child_frame_id = base_frame_id_;
+
     odom_pub_->msg_.pose.pose.position.z = 0;
-    odom_pub_->msg_.pose.covariance = boost::assign::list_of
-        (static_cast<double>(pose_cov_list[0])) (0)  (0)  (0)  (0)  (0)
```

Comment:
Should we change the param name of these covariance lists since they no longer mean what they used to?